std/file.d

```diff
@@ -4966,28 +4953,49 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
 }
 
 /// Duplicate functionality of D1's `std.file.listdir()`:
+version (StdDdoc)
 @safe unittest
 {
     string[] listdir(string pathname)
     {
-        import std.algorithm;
-        import std.array;
-        import std.file;
-        import std.path;
+        import std.algorithm.iteration : filter, map;
+        import std.array : array;
+        import std.path : baseName;
 
-        return std.file.dirEntries(pathname, SpanMode.shallow)
+        return dirEntries(pathname, SpanMode.shallow)
             .filter!(a => a.isFile)
-            .map!(a => std.path.baseName(a.name))
+            .map!(a => baseName(a.name))
             .array;
     }
 
-    void main(string[] args)
-    {
+    void main() {
         import std.stdio;
 
-        string[] files = listdir(args[1]);
-        writefln("%s", files);
-     }
+        string[] files = listdir("");
```

Comment:
1. Could use UFCS
2. Using `"."` makes it more clear that we're listing the contents of the current directory

```suggestion
        string[] files = ".".listdir();
```